package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "start": "webpack-dev-server --host 0.0.0.0 --hot",
     "build": "webpack -p",
-    "lint": "tslint -c tslint.json  -p tsconfig.json -t verbose 'src/**/*.{ts,tsx,js,jsx}'",
```

Comment:
Is this change intentional?

Reply:
Yeah, this is what was causing it to lint all the `.d.ts` files. It still lints regular files.